frontend/src/components/MapView/index.tsx

```diff
@@ -227,6 +227,8 @@ function MapView({ classes }: MapViewProps) {
     | LayerData<BoundaryLayerProps>
     | undefined;
 
+  // TODO - could we simply use the country boundary extent here instead of the calculation?
```

Comment:
@wadhwamatic this extent is used for the polygon analysis to limit the data to fetch. Currently we are calculating as the bbox of the boundary file geojson. Could we simply use the country bounds we are defining as part of this PR? Or can you think of edge cases where the admin boundaries span beyond what we would pass as the initial bounding box?

Reply:
I can't imagine a scenario where using the already defined country boundaries wouldn't work. Even for upcoming regional and global versions of PRISM where we have multiple countries, this should still work from what I understand.